tutorials/multimodal/NeVA Tutorial.ipynb

```diff
@@ -97,14 +94,15 @@
     " trainer.log_every_n_steps=1 \\\n",
     " trainer.max_steps=1000 \\\n",
     " model.megatron_amp_O2=True \\\n",
-    " model.micro_batch_size=1 \\\n",
-    " model.global_batch_size=2 \\\n",
+    " model.micro_batch_size=2 \\\n",
+    " model.global_batch_size=4 \\\n",
     " model.tensor_model_parallel_size=4 \\\n",
     " model.pipeline_model_parallel_size=1 \\\n",
     " model.mcore_gpt=True \\\n",
     " model.transformer_engine=True \\\n",
     " model.data.data_path=/path/to/datasets/LLaVA-Pretrain-LCS-558K/blip_laion_cc_sbu_558k.json \\\n",
```

Comment:
add a section before this running command to link users to prepare all dataset and required checkpoints from https://docs.nvidia.com/nemo-framework/user-guide/latest/multimodalmodels/multimodallanguagemodel/neva/dataprep.html